docs/tutorials/index.rst

```diff
@@ -348,3 +348,17 @@ Deployment
     ../build/examples_datasets/index
     ../build/examples_distributed/index
     ../build/examples_deployment/index
+    ../build/examples_tracking/index
+
+
+
+Object Tracking
```

Comment:
Since object tracking is an application, it is better to put it right after video action recognition, and before dataset preparation. `doctree` should always be the last thing of this page.